src/transaction/flashback.c

```diff
@@ -189,7 +189,8 @@ flashback_make_summary_list (THREAD_ENTRY * thread_p, FLASHBACK_SUMMARY_CONTEXT
 
 	    if (context->num_summary == FLASHBACK_MAX_NUM_TRAN_TO_SUMMARY)
 	      {
-		/* er_set */
+		er_set (ER_NOTIFICATION_SEVERITY, ARG_FILE_LINE, ER_FLASHBACK_EXCEED_MAX_NUM_TRAN_TO_SUMMARY, 1,
```

Comment:
As I know, the "error_log_level" system parameter usually is not set to NOTIFICATION in the environment of our customer. So I recommend you to use _er_log_debug or to change the error log level. Otherwise, you may not get any debugging information when the problem occurs.

Reply:
What I saw in manual and in code, 
system parameter (PRM_ID_ER_LOG_LEVEL) is set to ER_NOTIFICATION_SEVERITY by default. 
So, I thought it would be normal for customers to leave this parameter unchanged.